lib/common.sh

```diff
@@ -14,13 +14,18 @@ DAPP_LIB=${DAPP_LIB:-$BIN_DIR/contracts}
 # Declare functions
 
 setConfigFile() {
-    # Clean out directory
-    rm -rf "$OUT_DIR" && mkdir "$OUT_DIR"
-    # If environment variable exists bring the values from there, otherwise use the config file
-    if [[ -n "$TDDS_CONFIG_VALUES" ]]; then
-        echo "$TDDS_CONFIG_VALUES" > "$CONFIG_FILE"
-    else
-        cp "$CONFIG_DIR/deploy-$1.json" "$CONFIG_FILE"
+    # Check if we have already set the config values
+    if [[ "$TDDS_CONFIG_VALUES_SET" != "true" ]]; then
```

Comment:
Are the changes to this file necessary for the PR or just improvements?

If the latter, I'd maybe file them in a separate PR.

Reply:
This was to allow "stacking" steps.  Without this change when the `setup` script called the `kovan` script, it would overwrite the config file